com.unity.ml-agents/Runtime/Sensors/IBuiltInSensor.cs

```diff
@@ -0,0 +1,37 @@
+namespace Unity.MLAgents.Sensors
+{
+    /// <summary>
+    /// Identifiers for "built in" sensor types.
+    /// These are only used for analytics, and should not be used for any runtime decisions.
+    /// </summary>
+    public enum BuiltInSensorType
+    {
+        Unknown = 0,
+        VectorSensor = 1,
+        // Note that StackingSensor actually returns the wrapped sensor's type
+        StackingSensor = 2,
+        RayPerceptionSensor = 3,
+        ReflectionSensor = 4,
+        CameraSensor = 5,
+        RenderTextureSensor = 6,
+        BufferSensor = 7,
+        PhysicsBodySensor = 8,
```

Comment:
The ordering doesn't actually matter here, but I could make the extensions implementations start at some offset, say 1000.